modules/genomic_browser/test/TestPlan.md

```diff
@@ -1,24 +1,24 @@
-#Genomic Browser module - Test Plan
-<br>
-###Table of Contents  
-
-####Permissions   
-####Navigation
-####Features
-######[Filtering](#data_filtering)  
-######[Datatable](#datatable)
-######[Download](#data_download)  
-######[Upload](#data_upload)  
-####Help
-######[Help content](#help_content)
-######[Tooltips](#tooltips)
+## Genomic Browser - Test Plan
+
+## Table of Contents  
+
+### [Permissions](#permissions)
```

Comment:
@xlecours If you tell me the changes to make I will update the PR since @llevitis is not here